### PR TITLE
[Privilege] Deprecate privacy related API

### PR DESCRIPTION
**Pull request by yl33**
### Description of Change ###
- Remove privacy related API

### API Changes ###

 - ACR: https://code.sec.samsung.net/jira/browse/TCSACR-524

### CODE REVIEW

File: src/Tizen.Security/Tizen.Security/Privilege.cs [hunk 1]
```diff
@@ -153,6 +153,7 @@ public static string GetDescription(string apiVersion, string privilege, Package
         /// <exception cref="System.OutOfMemoryException">Thrown when out of memory occurs.</exception>
         /// <exception cref="System.InvalidOperationException">Thrown when an internal error occurs.</exception>
         /// <exception cref="System.NotSupportedException">The required feature is not supported.</exception>
+        [Obsolete("Deprecated since API level 11.")]
```
Comment (wiertel) [inline src/Tizen.Security/Tizen.Security/Privilege.cs:156]: it's better to include information about replacement, is there any a way a user can achieve similar result with other API?

Comment (earth-table-coffee) [reply src/Tizen.Security/Tizen.Security/Privilege.cs:156]: There's no replacement of it. Would it be better to add 'Please don't use it' message?

Comment (yl33) [reply src/Tizen.Security/Tizen.Security/Privilege.cs:156]: There' no replacement of it. Would it be better to add 'Please don't use it' message instead?

**Review by kd0228-kim — APPROVED** [review]

File: src/Tizen.Security/Tizen.Security/Privilege.cs (hunk 1 above)
Comment (yl33) [reply src/Tizen.Security/Tizen.Security/Privilege.cs:156]: @wiertel Please review this PR.

**Comment by yl33** [comment]
something wrong with auto rebase, will open new PR

**Pull request closed without merging**